geoips/create_plugin_registries.py

```diff
@@ -225,6 +225,12 @@ def registry_sanity_check(plugin_packages, save_type):
                                             # the package Product plugin,
                                             # raise a PluginRegistryError
                                             # and remove the registries.
+                                            pkg_relpath = "not defined"
+                                            if "relpath" in pkg_plugin:
+                                                pkg_relpath = pkg_plugin["relpath"]
+                                            subplg_relpath = "not defined"
+                                            if "relpath" in sub_plg:
+                                                subplg_relpath = sub_plg["relpath"]
```

Comment:
This avoids getting an unrelated useless error (when relpath is not in pkg_plugin...).  Better error messages